packages/lab/src/progress/CircularProgress/CircularProgress.tsx

```diff
@@ -10,20 +9,21 @@ import circularProgressCSS from "./CircularProgress.css";
 
 const withBaseName = makePrefixer("saltCircularProgress");
 
-export interface CircularProgressProps extends HTMLAttributes<HTMLDivElement> {
+export interface CircularProgressProps extends ComponentPropsWithoutRef<"div"> {
   /**
-   * A label for accessibility
+   * Whether to hide the text label within the progress. Defaults to `false`.
    */
-  "aria-label"?: string;
-  /**
-   * The className(s) of the component
-   */
-  className?: string;
+  hideLabel?: boolean;
```

Comment:
Can we default `children` to `Text` instead of `hideLabel` to offer better flexibility / composability?

Reply:
We had a chat about this in the KO, but it is not the plan, specially because of circular's limited space, to allow users to pass anything that might overflow into the component.